build-parent/pom.xml

```diff
@@ -166,6 +166,29 @@
         </dependencies>
     </dependencyManagement>
 
+    <dependencies>
+        <!-- START: workaround to get maven build cache invalidation on new SNAPSHOTS of commonly updated plugins:  -->
+        <dependency>
+            <groupId>org.technologybrewery.fermenter</groupId>
+            <artifactId>fermenter-mda</artifactId>
+            <version>${version.fermenter}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.technologybrewery.habushu</groupId>
+            <artifactId>habushu-maven-plugin</artifactId>
+            <version>${version.habushu.plugin}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.technologybrewery.baton</groupId>
+            <artifactId>baton-maven-plugin</artifactId>
+            <version>${version.baton}</version>
+            <scope>provided</scope>
+        </dependency>
+        <!-- END: workaround to get maven build cache invalidation on new SNAPSHOTS of commonly updated plugins -->
+    </dependencies>
+
```

Comment:
How does this workaround fix the issue where new versions of SNAPSHOTS are not being invalidated? It looks like the **version.fermenter** is a SNAPSHOT version.

Reply:
Please see the comment above: https://github.com/boozallen/aissemble/pull/160/files#r1649063234